flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/slot/TaskSlotTableImpl.java

```diff
@@ -134,6 +143,12 @@ public TaskSlotTableImpl(
 		slotActions = null;
 		state = State.CREATED;
 		closingFuture = new CompletableFuture<>();
+
+		asyncExecutor = new ThreadPoolExecutor(
```

Comment:
All executors should use a `org.apache.flink.runtime.util.ExecutorThreadFactory`.

Reply:
I suppose mostly for naming purposes?
`Util.threadFactory("tm-slot-async-call", false)`

Reply:
No. Mainly for having an uncaught exception handler configured.